filemanager/parts.py

```diff
@@ -797,6 +815,11 @@ def kind_of_part_protomodule(self):
 		if not tmp_proto.load(self.protomodule):  return None
 		return tmp_proto.kind_of_part
 
+	@property
+	def barcode(self):
+		if self.ID is None:
+			return None
+		return "320" + self.ID.replace("-", "")
```

Comment:
will everything start with 320 in the future too?

Reply:
Yes, the 320 is the project code for HGCAL. With correct module naming scheme, the barcode will just be 320+serial number (ID)